cypress/integration/taskCloneTableColumn.spec.js

```diff
@@ -1,33 +1,24 @@
 'use strict';
 
 describe('Clone table app', () => {
-  before(() => {
+  beforeEach(() => {
     cy.visit('/');
+    cy.get('th:nth-child(2)').debug();
```

Comment:
The use of `cy.debug()` is generally for debugging purposes and should not be included in the final test code as it may cause confusion or clutter the test output. Consider removing it unless it's necessary for the test.